lib/libvcc/vcc_vmod.c

```diff
@@ -106,6 +99,21 @@ vcc_ParseImport(struct vcc *tl)
 	}
 
 	SkipToken(tl, ';');
+	if (osym != NULL) {
+		AN(osym->path);
+		if (strcmp(osym->path, fn) != 0) {
+			VSB_printf(tl->sb, "Warning: Module %.*s imported from different location.\n",
+			    PF(mod));
+			vcc_ErrWhere2(tl, t1, tl->t);
+			VSB_printf(tl->sb, "Previous import was here:\n");
+			vcc_ErrWhere2(tl, osym->def_b, osym->def_e);
+			tl->err = 0; // reset error; this is just a warning
```

Comment:
> There is a use case; let's say you ship a vmod_zoo and zaa.vcl, and you want to test it without installing the vmod; then you either have to modify the vcl to say "import zoo from bar" or you can, in your vtc specify
>   import zoo from "/path/to/bar"; include "zaa.vcl";
> [..]

That's a reasonable argument although I'm wondering how common it is and whether allowing for this use case won't affect users, as in not erroring out when we should.

I'm starting to think we might need a more granular error/warning way to express this, similar to -p debug, e.g., -p vcc_error +unref +import.